truss/contexts/image_builder/cache_warmer.py

```diff
@@ -32,21 +32,36 @@ def _download_from_url_using_b10cp(
     )
 
 
+def split_gs_path(gs_path):
+    # Remove the 'gs://' prefix
+    path = gs_path.replace("gs://", "")
+
+    # Split on the first slash
+    parts = path.split("/", 1)
+
+    bucket_name = parts[0]
+    prefix = parts[1] if len(parts) > 1 else ""
+
+    return bucket_name, prefix
+
+
 def download_file(
     repo_name, file_name, revision_name=None, key_file="/app/data/service_account.json"
 ):
     # Check if repo_name starts with "gs://"
     if "gs://" in repo_name:
         # Create directory if not exist
+        bucket_name, _ = split_gs_path(repo_name)
```

Comment:
i wonder if it's better to strip the repo_name when templating out the command we're passing in instead of parsing it differently here.